open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/eventprocessor/EventPublisher.java

```diff
@@ -329,7 +331,8 @@ public void processConflictingTypeEvent(String sourceName,
      * @param event to be published
      * @return true/false based on the success of the operation
      */
-    public boolean sendEvent(DataPlatformEventHeader event) {
+    public boolean sendEvent(NewViewEvent event) {
```

Comment:
It makes sense to keep the parent abstract class as argument, it should work the same for any event type.

Reply:
yes somehow I misunderstood this part. Will change it.